wd-java/src/test/java/com/example/InventoryFullPageTest.java

```diff
@@ -0,0 +1,68 @@
+package com.example;
+
+import com.example.pageobjects.InventoryLongPage;
+import com.example.pageobjects.LoginPage;
+import com.saucelabs.visual.CheckOptions;
+import com.saucelabs.visual.DataCenter;
+import com.saucelabs.visual.VisualApi;
+import com.saucelabs.visual.junit5.TestMetaInfoExtension;
+import com.saucelabs.visual.model.FullPageScreenshotConfig;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.openqa.selenium.remote.RemoteWebDriver;
+
+import java.net.MalformedURLException;
+
+import static com.example.TestUtils.dotenv;
+
+
+@ExtendWith({TestMetaInfoExtension.class})
+public class InventoryFullPageTest {
+
+    private static final String username = dotenv.get("SAUCE_USERNAME");
+    private static final String accessKey = dotenv.get("SAUCE_ACCESS_KEY");
+
+    private static VisualApi visual;
+    private static RemoteWebDriver driver;
+
+    @BeforeAll
+    public static void init() throws MalformedURLException {
+        driver = TestUtils.getDriver(username, accessKey);
+        visual = new VisualApi.Builder(driver, username, accessKey, DataCenter.US_WEST_1)
+                .withBuild("Sauce Demo Test")
+                .withBranch("main")
+                .withProject("JUnit + WebDriver examples for " + System.getenv("SAUCE_USERNAME"))
+                .withCaptureDom(true)
+                .build();
+    }
+
+    @Test
+    void checkInventoryPageLooksTheSame() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.open();
+
+        visual.sauceVisualCheck("Before Login");
+        String user = "standard_user";
+        if (System.getProperty("modified") != null || dotenv.get("VISUAL_CHECK") != null) {
```

Comment:
We can probably omit this check for `modified` and `VISUAL_CHECK` for this test

Reply:
Then it will be no diffs for full page screenshots. I would like to have diffs for fps too.

Reply:
Ok let's keep only `modified` then. VISUAL_CHECK is not used in java example yet